app/services/youtube_service/client.rb

```diff
@@ -0,0 +1,38 @@
+class YoutubeService
+  class Client
+    API_PREFIX = 'https://www.googleapis.com/youtube/v3'.freeze
+    CHANNEL_URL = Addressable::Template.new(
+      "#{API_PREFIX}/channels.list{?query*}"
+    ).freeze
+    VERIFY_URL = Addressable::Template.new(
+      "#{API_PREFIX}/tokeninfo{?query*}"
+    ).freeze
```

Comment:
Looks like the URL here is actually `https://www.googleapis.com/oauth2/v3/tokeninfo`